batch/Dockerfile.worker

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-slim-stretch
+FROM {{ hail_ubuntu_image.image }}
```

Comment:
Can you compare the sizes of both images? I'm worried it will be substantially bigger and will affect worker startup time.

Reply:
The Hail Ubuntu adds about 500MB. The worker image was 300MB previously. I switched it back to the python image.